packages/board-server/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build stage
-FROM node:20-slim AS build
```

Comment:
darwin does not exit for node images
see:
- [node:20-slim](https://hub.docker.com/layers/library/node/20-slim/images/sha256-139024e06dec0692a40598f091497fe9c75af507f6f3685858be00d3312b1dcb)
- [node:20]( https://hub.docker.com/layers/library/node/20/images/sha256-139024e06dec0692a40598f091497fe9c75af507f6f3685858be00d3312b1dcb)